IllegalArgumentException: Provided length 32 is too big

```
2020-08-05 05:19:20.726+00:00 | nioEventLoopGroup-4-1 | WARN  | DefaultChannelPipeline | An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
reactor.core.Exceptions$ErrorCallbackNotImplemented: java.lang.IllegalArgumentException: Provided length 32 is too big: the value has size 18 and has only 18 bytes from 0
Caused by: java.lang.IllegalArgumentException: Provided length 32 is too big: the value has size 18 and has only 18 bytes from 0
        at com.google.common.base.Preconditions.checkArgument(Preconditions.java:478) ~[guava-29.0-jre.jar:?]
        at org.apache.tuweni.bytes.ArrayWrappingBytes.slice(ArrayWrappingBytes.java:74) ~[tuweni-bytes-1.0.0.jar:1.0.0]
        at org.ethereum.beacon.discovery.packet.UnknownPacket.getSourceNodeId(UnknownPacket.java:51) ~[discovery-0.3.7.jar:0.3.7]
        at org.ethereum.beacon.discovery.pipeline.handler.UnknownPacketTagToSender.handle(UnknownPacketTagToSender.java:49) ~[discovery-0.3.7.jar:0.3.7]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:189) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:203) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:203) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:203) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replayNormal(FluxReplay.java:814) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replay(FluxReplay.java:898) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.ReplayProcessor.onNext(ReplayProcessor.java:442) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxCreate$IgnoreSink.next(FluxCreate.java:618) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxCreate$SerializedSink.next(FluxCreate.java:153) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at org.ethereum.beacon.discovery.pipeline.PipelineImpl.push(PipelineImpl.java:43) ~[discovery-0.3.7.jar:0.3.7]
        at reactor.core.publisher.LambdaSubscriber.onNext(LambdaSubscriber.java:160) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replayNormal(FluxReplay.java:814) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replay(FluxReplay.java:898) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.ReplayProcessor.onNext(ReplayProcessor.java:442) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxCreate$IgnoreSink.next(FluxCreate.java:618) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at reactor.core.publisher.FluxCreate$SerializedSink.next(FluxCreate.java:153) ~[reactor-core-3.3.7.RELEASE.jar:3.3.7.RELEASE]
        at org.ethereum.beacon.discovery.network.IncomingMessageSink.channelRead0(IncomingMessageSink.java:31) ~[discovery-0.3.7.jar:0.3.7]
        at org.ethereum.beacon.discovery.network.IncomingMessageSink.channelRead0(IncomingMessageSink.java:20) ~[discovery-0.3.7.jar:0.3.7]
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) ~[netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:271) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.nio.AbstractNioMessageChannel$NioMessageUnsafe.read(AbstractNioMessageChannel.java:93) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.51.Final.jar:4.1.51.Final]
        at java.lang.Thread.run(Thread.java:830) [?:?]
```

Version: teku/v0.12.3-dev-503652c9/linux-x86_64/-privatebuild-openjdk64bitservervm-java-13

--- a/gradle/versions.gradle
+++ b/gradle/versions.gradle
@@ -137,7 +137,7 @@ dependencyManagement {
     dependency "org.testcontainers:testcontainers:1.14.3"
     dependency "org.testcontainers:junit-jupiter:1.14.3"
 
-    dependency 'tech.pegasys.discovery:discovery:0.3.7'
+    dependency 'tech.pegasys.discovery:discovery:0.3.8-dev-9236557c'
 
     dependency 'tech.pegasys.signers.internal:bls-keystore:1.0.1'
   }

--- a/gradle/versions.gradle
+++ b/gradle/versions.gradle
@@ -137,7 +137,7 @@ dependencyManagement {
     dependency "org.testcontainers:testcontainers:1.14.3"
     dependency "org.testcontainers:junit-jupiter:1.14.3"
 
-    dependency 'tech.pegasys.discovery:discovery:0.3.8-dev-9236557c'
+    dependency 'tech.pegasys.discovery:discovery:0.3.8-dev-840e90be'
 
     dependency 'tech.pegasys.signers.internal:bls-keystore:1.0.1'
   }

